### ISSUE TITLE
DST Staging Review: Alerts are not announced

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#6133]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** 
**Staging Review ticket:** [Staging Review: va-file-input with password](https://github.com/department-of-veterans-affairs/va.gov-team/issues/138797)

## Findings details
**VA.gov Experience Standard - issue:** [User can perceive all meaningful elements.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**This is an issue with the:** code
**High-priority:** yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

The info and success alerts are not announced to assistive tech.
In setting "submission status - success" on the file upload, the content "File successfuly unlocked" is appended to the screen dynamically, and it is not announced to assistive technology. Likewise, the warning alert, "We can't open your file without its password" is not announced and is not associated with the "File password" field, meaning that screen reader users may struggle with interacting with this component. 

Tested:
- Jaws/Chrome
- Jaws/Edge
- Mac/VO/Safari

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="1381" height="495" alt="alerts don't announce" src="https://github.com/user-attachments/assets/f0866c58-ee51-4c63-8122-bad2c7184d71" />

</details>

## Recommended action
Ensure that the status messages are announced, as they are feedback to a user action, and without active announcement, assistive technology users may miss this content - especially because the alerts are not announced on focus on the component. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** Screen reader
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by tbaker1026** [comment]
Jeana, Amy & Ryan to discuss how this will work.

**Comment by tbaker1026** [comment]
Should have a PR open soon
